src/solvers/dg/2d/parallel.jl

```diff
@@ -0,0 +1,716 @@
+# Calculate time derivative
+function rhs!(dg::Dg2D, t_stage, uses_mpi::Val{true})
+  # Start to receive MPI data
+  @timeit timer() "start MPI receive" start_mpi_receive!(dg)
+
+  # Reset u_t
+  @timeit timer() "reset ∂u/∂t" dg.elements.u_t .= 0
```

Comment:
It probably doesn't cost much time but should this be moved after `start_mpi_send!` to overlap more communication and computation?

Reply:
Fixed in 3c392ca.